ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -1,10 +1,12 @@
+{-# LANGUAGE FlexibleContexts    #-}
```

Comment:
There are no actual changes in this module, I think all the changes in this module can be dropped from the PR.

Reply:
Done!